tracer/build/_build/Build.Utilities.cs

```diff
@@ -356,6 +358,47 @@ private void ReplaceReceivedFilesInSnapshots()
         }
     }
 
+    private async Task<bool> IsDebugRun()
+    {
+        var forceDebugRun = Environment.GetEnvironmentVariable("ForceDebugRun");
```

Comment:
Not related per se to this change, but we could maybe also add a build variable in the pipeline in case we want to run a debug run manually. We could check the variable here, so no extra change would be required. WDYT?

Reply:
> but we could maybe also add a build variable in the pipeline

I already added `ForceDebugRun` as a variable to the pipeline so we can force a debug manually, that's how I ran [this one](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=136701&view=results). Is that what you mean? There's no need for an extra check AFAIK

Reply:
Oh neat, I thought it had to be defined in the yml. :til: